content/07-native-tokens/03-getting-started.mdx

```diff
@@ -49,49 +49,52 @@ Once you have started the node, leave it running in its own terminal window (or
 Once a relay node has been set up, the `cardano-cli` command can be used to interact with the network as usual: 
 
 ```
-cd mary-ppe
+cd testnet
 cardano-cli ...
 ```
 
-Note that multi-asset support is provided as part of the Mary consensus era, so many commands will require the `--mary-era` flag:
+### Understanding values
+
+cardano-node version 1.35.0 and later of the cli no longer supports ASCII token names, only hex-encoded format is supported. The result of running  
+
 
 ```
-cardano-cli ... --mary-era
-```
+$ echo -n "assetName" | xxd -ps
 
-### Understanding values
+```
 
 Lovelace values can be specified in two ways:
 
 - `${quantity} lovelace` (where quantity is a signed integer)
 - `${quantity}` (where quantity is a signed integer)
+- `${assetName}` (where assetName is hex-encoded assetName)
 
 Values for other assets can be specified as:
 
 - `${quantity} ${policyId}.${assetName}`
 - `${quantity} ${policyId}` 
 
-Where `quantity` is a signed integer and `policyId` is a hex-encoded policy ID [a script hash]), and `assetName` is an alphanumeric asset name.
+Where `quantity` is a signed integer and `policyId` is a hex-encoded policy ID [a script hash]), and `assetName` is a hex-encoded assetName.
 
 #### Syntax of multi-asset values
 
 The `cardano-cli` can specify multi-asset values in transaction outputs and when minting or burning tokens. The syntax for these values has been designed to be backwards-compatible with the previous ada-only syntax (`address+lovelace`):
 
 - ada values are defined as integer (INT) lovelace, e.g. `42 lovelace`
 - multi-asset values can be defined as:
-  - `INT policyid.assetname`, e.g. `42 $MYPOLICY.myassetname`
+  - `INT policyid.assetName`, e.g. `42 $MYPOLICY.assetName`
   - `INT policyid`, e.g. `42 $MYPOLICY` (No asset name specified)
-  - `policyid.assetname`, e.g `$MYPOLICY.myassetname` (This will mint only one of `myassetname`)
+  - `policyid.assetName`, e.g `$MYPOLICY.assetName` (This will mint only one of `myassetname`)
```

Comment:
As per my previous comment, we should use examples with hexadecimal values
and
Pretty sure you always need to specify the number of assets
so
"e.g `1 $MYPOLICY.61737365744e616d65`"

Reply:
`myassetname` is now referred to as `assetName` so change all instances.